helm/dashboards/dashboards/shared/private/loki-cost-estimation.json

```diff
@@ -42,45 +42,44 @@
   "liveNow": false,
   "panels": [
     {
-      "collapsed": true,
+      "collapsed": false,
       "gridPos": {
         "h": 1,
         "w": 24,
         "x": 0,
         "y": 0
       },
       "id": 41,
-      "panels": [
-        {
-          "datasource": {
-            "type": "prometheus",
-            "uid": "PBFA97CFB590B2093"
-          },
-          "description": "",
-          "gridPos": {
-            "h": 16,
-            "w": 24,
-            "x": 0,
-            "y": 1
-          },
-          "id": 43,
-          "options": {
-            "code": {
-              "language": "plaintext",
-              "showLineNumbers": false,
-              "showMiniMap": false
-            },
-            "content": "This dashboard is a tool to estimate the cost of Loki.\n\n`loki` is deployed on the MC and `promtail` is deployed on each WC + MC.\n\n`promtail` sends the logs to `loki` and `loki` sends the data on `S3`.\n\nTo evaluate the real cost of `loki`, we need to take in account:\n- CPU and memory used by all loki pods\n- data sent by all WC promtail to loki\n- data sent on S3\n\n\nOnly the traffic between EC2 instances and Internet has a cost. <br/>\nTo estimate the global cost of the traffic for `loki`, we have to mesure the traffic between `promtail` pods on WC and `loki` on MC.<br/>\nThe traffic between `promtail` pods on MC and `loki` is free.\n\nTotal of bytes stored on S3 is estimated from data sent to S3. Hence, we are not aware of data cleaning.\n\nA tenant is related to a cluster. `tenant` selection allows you to know:\n- how much data is sent on S3 for the selected tenants\n- how many logs are sent by the selected tenants\n\n\n`cluster` selection allows you to know:\n- how much traffic is transmitted between WC and Loki (you have to select all the clusters in the list except the MC)\n- how many pods are running on the selected clusters",
-            "mode": "markdown"
-          },
-          "pluginVersion": "10.0.3",
-          "title": "Explanation",
-          "type": "text"
-        }
-      ],
+      "panels": [],
```

Comment:
A panel got emptied?

Reply:
In any case I tried this dashboard by importing it as a new one and it works as intended; Is that ok for you @hervenicol ?

Reply:
I guess the panel has changed placeholder, but I don't know exactly what happened.
A before/after screenshot would have helped understand what has changed or not :grin: